Champions/MasterYi/R.cs

```diff
@@ -26,11 +27,11 @@ public void OnStartCasting(Champion owner, Spell spell, AttackableUnit target)
         {
             var p = ApiFunctionManager.AddParticleTarget(owner, "Highlander_buf.troy", target, 1);
             var buff = ((ObjAiBase) target).AddBuffGameScript("Highlander", "Highlander", spell);
-            var visualBuff = ApiFunctionManager.AddBuffHudVisual("Highlander", 10.0f, 1, owner);
+            var visualBuff = ApiFunctionManager.AddBuffHudVisual("Highlander", 10.0f, 1, BuffType.COMBAT_ENCHANCER,
+                owner, 10.0f);
             ApiFunctionManager.CreateTimer(10.0f, () =>
             {
                 ApiFunctionManager.RemoveParticle(p);
-                ApiFunctionManager.RemoveBuffHudVisual(visualBuff);
```

Comment:
Is there a downside to not removing the buff hud visual?

Reply:
See above comment regarding the same question in regards to SummonerRevive. I should probably use AddBuffHudVisual()'s Timer argument in all applicable cases for completion, though.